packages/dd-trace/test/ci-visibility/exporters/git/git_metadata.spec.js

```diff
@@ -202,7 +202,57 @@ describe('git_metadata', () => {
       done()
     })
   })
+  it('should return false if Git repository URL is invalid', (done) => {
```

Comment:
`done` is only used when tests are asynchronous, but this is not the case here, so we can remove it 
```suggestion
  it('should return false if Git repository URL is invalid', () => {
```